monochart/nginx-config-patch-monochart-testing.yaml

```diff
@@ -0,0 +1,19 @@
+spec:
```

Comment:
Was this file intended to be included? Looks like it's for testing unless I'm missing something.

Reply:
Maybe move it into a `patches/` subdirectory would make its intent more clear?

Reply:
nope, not supposed to be included, sorry thought I got them all 😂

Reply:
removed.